docs/docs/examples/logging/efk/EFK.md

```diff
@@ -75,51 +60,62 @@ fluentd:
+Through port-forwarding you can access Kibana via `http://localhost:5601`
+```shell
+kubectl port-forward deployment/kibana-kibana 5601
+```
+ To visualise the logs you need to create an index pattern and then look at the the data in the discovery part. To create the index pattern go to `Management` → `Stack Management` and then select `Kibana` → `Index Patterns`. 
 
-## Managed Elasticsearch in AWS
+## Managed EFK stack
 
-In this solution Elasticsearch deploys as a managed service and lives outside of the Kubernetes cluster. For this purpose use Fluent Bit instead of Fluentd for local deployment of EFK. 
+In this solution [Elasticsearch is deployed as a managed AWS service](https://aws.amazon.com/elasticsearch-service/){.external} and lives outside of the Kubernetes cluster. This approach uses [fluentBit](https://fluentbit.io/){.external} instead of `Fluentd` for log processing.
 
-When a node inside an EKS cluster needs to call an AWS API, it needs to provide extended permissions. Amazon provides an image of Fluent Bit that supports AWS service accounts,and using this you no longer need to follow the traditional way. All you need is to have an IAM role for the AWS service account on an EKS cluster. So using this service account, an AWS permission can be provided to the containers in any pod that use that service account. The result is that the pods on that node can call AWS APIs.
+???+ info "Fluentbit"
 
-`fluentbit` is used to collect and aggregate the data inside the EKS cluster, which communicates with AWS Elasticsearch outside of the cluster. 
+    `Fluentbit` is used to collect and aggregate log data inside the EKS cluster. It then sends this to an AWS Elasticsearch instance outside of the cluster.
 
-Your first step is to configure IAM roles for Service Accounts (IRSA) for `fluentbit`, to make sure you have an OIDC identity provider to use IAM roles for the service account in the cluster:
+
+When a node inside an EKS cluster needs to call an AWS API, it needs to provide extended permissions. Amazon provides an image of `Fluentbit` that supports AWS service accounts,and using this you no longer need to follow the traditional way. All you need is to have an IAM role for the AWS service account on an EKS cluster. Using this service account, an AWS permission can be provided to the containers in any pod that use that service account. The result is that the pods on that node can call AWS APIs.
+
+Your first step is to configure IAM roles for Service Accounts (IRSA) for `Fluentbit`, to make sure you have an OIDC identity provider to use IAM roles for the service account in the cluster:
 
 ```shell
-$ eksctl utils associate-iam-oidc-provider \
+eksctl utils associate-iam-oidc-provider \
      --cluster dcd-ap-southeast-2 \
```

Comment:
Maybe good to change cluster name to something else ( `<cluster-name>` or `my-k8s-cluster` ). WDYT?

Reply:
changes to `<cluster-name>`